src/method.js

```diff
@@ -0,0 +1,173 @@
+import Lists from './listClass.js';
+
+export default class MainFunc {
```

Comment:
I think it would be better that your function name is self descriptive instead of MainFunc you could use MainFunction.

Reply:
Thank you @lugard1 for reviewing my pull request 👍